lib/Sema/TypeCheckMacros.cpp

```diff
@@ -367,33 +364,28 @@ loadExecutablePluginByName(ASTContext &ctx, Identifier moduleName) {
 LoadedCompilerPlugin
 CompilerPluginLoadRequest::evaluate(Evaluator &evaluator, ASTContext *ctx,
                                     Identifier moduleName) const {
-  auto fs = ctx->SourceMgr.getFileSystem();
-  auto &searchPathOpts = ctx->SearchPathOpts;
-  auto *registry = ctx->getPluginRegistry();
```

Comment:
Unrelated changes but these variables weren't used. I should have removed this in d8c8574e9bb01b23a7440de8cdac401d584ef667